chapters/Index.md

```diff
@@ -35,24 +35,2394 @@ Jintek           <info@jintek.consulting>
+
+A conforming **did:btcr2** resolver is any algorithm realized as software and/or hardware that complies with the relevant normative statements in [4\. DID Resolution](https://www.w3.org/TR/did-resolution/#resolving) of the DID Resolution specification and the \[Read\] section of this specification.
+
+## Terminology
+
+Aggregate Beacon
+
+: An Aggregate Beacon enables multiple entities (possibly controlling multiple DIDs and possibly posting multiple updates) to collectively announce a set of ::BTCR2 Updates:: in a ::Beacon Signal::.
+
+There can only ever be one ::BTCR2 Update:: per **did:btcr2** in a ::Beacon Signal:: from an Aggregate Beacon.
+
+Aggregate Beacons
+
+: ::Aggregate Beacon::
+
+Announced BTCR2 Update
+
+: A ::BTCR2 Update:: that has been announced in an ::Authorized Beacon Signal:: which has met the specified threshold for confirmation.
+
+Announced BTCR2 Updates
+
+: ::Announced BTCR2 Update::
+
+Authorized Beacon Signal
+
+: An Authorized Beacon Signal is a ::Beacon Signal:: from a ::BTCR2 Beacon:: with a ::BTCR2 Beacon:: address in a then-current DID document.
+
+Authorized Beacon Signals
+
+: ::Authorized Beacon Signal::
+
+Beacon Address
+
+: The Bitcoin address of a ::BTCR2 Beacon::. Spends of ::UTXO:: controlled by this address are identified as ::Beacon Signals::.
+
+Beacon Aggregator
+
+: The entity that coordinates the protocols of an aggregate ::BTCR2 Beacon::. Specifically the Create Beacon Cohort and Announce Beacon Signal protocols.
+
+Beacon Aggregators
+
+: ::Beacon Aggregator::
+
+Beacon Announcement Map
+
+: A document that maps **did:btcr2** identifiers to ::BTCR2 Update Announcements::. This is used to distinguish which ::BTCR2 Update Announcement:: applies to which **did:btcr2** identifier.
+
+Beacon Announcement Maps
+
+: ::Beacon Announcement Map::
+
+Beacon Cohort
+
+: The set of unique cryptographic keys participating in a ::BTCR2 Beacon:: that are required to authorize spends from the ::Beacon Address::.
+
+Beacon Cohorts
+
+: ::Beacon Cohort::
+
+Beacon Participant
+
+: A member of a ::Beacon Cohort::, typically a DID controller, that controls cryptographic keys required to partially authorize spends from a ::Beacon Address::.
+
+Beacon Participants
+
+: ::Beacon Participant::
+
+Beacon Signal
+
+: Beacon Signals are Bitcoin transactions that spend from a ::BTCR2 Beacon:: address and include a transaction output of the format `[OP_RETURN, <32_bytes>]`. Beacon Signals announce one or more ::BTCR2 Updates:: and provide a means for these payloads to be verified as part of the Beacon Signal.
+
+The type of the ::BTCR2 Beacon::  determines how these Beacon Signals SHOULD be constructed and processed to validate a set of ::BTCR2 Updates:: against the 32 bytes contained within the Beacon Signal.
```

Comment:
```suggestion
The type of the ::BTCR2 Beacon::  determines how these Beacon Signals MUST be constructed and processed to validate a set of ::BTCR2 Updates:: against the ::Signal Bytes:: contained within the Beacon Signal.
```

Not sure if we want MUST in a terminology, but it shouldnt be SHOULD I dont think. Maybe just remove entire sentence?